src/OrchardCore.Modules/OrchardCore.Contents/Shapes.cs

```diff
@@ -5,9 +5,11 @@
 using OrchardCore.DisplayManagement;
 using OrchardCore.DisplayManagement.Descriptors;
 using OrchardCore.DisplayManagement.ModelBinding;
+using OrchardCore.Modules;
 
 namespace OrchardCore.Contents
 {
+    [Feature("OrchardCore.Contents")]
```

Comment:
Need to be tried but I think the feature attribute is not needed when related to the module main feature.

Reply:
done